external-crates/move/move-compiler/src/typing/translate.rs

```diff
@@ -1419,6 +1419,11 @@ fn exp_inner(context: &mut Context, sp!(eloc, ne_): N::Exp) -> T::Exp {
             (ty, TE::ExpList(items))
         }
         NE::Pack(m, n, ty_args_opt, nfields) => {
+            // all fields of a packed struct type are used
+            let used_fields: BTreeSet<Symbol> =
+                BTreeSet::from_iter(nfields.iter().map(|(_, s, _)| *s));
+            context.used_fields.insert(n.value(), used_fields);
```

Comment:
You probably don't want to accidentally clobber what was there.  I know pack _should_ have all of the fields, but it might not if there was an error, so it will be potentially annoying to get a unused field warning because of a bad pack somewhere